versions.yaml

```diff
@@ -221,11 +221,21 @@ externals:
     url: "https://github.com/kubernetes-sigs/cri-tools"
     version: "1.23.0"
 
+  cryptsetup:
+    description: "A utility used to setup disk encryption, integrity protection"
+    url: "https://gitlab.com/cryptsetup/cryptsetup"
+    version: "v2.5.0"
+
```

Comment:
Asking here, but it applies to previous parts of this commit.

Do you need cryptsetup and lvm2 in order to generate the initramfs for SEV?  Or is this being pulled in just because it was part of the root_hash verification work?

If that's the latter, drop it and the related parts, as that should come in a different PR.

Reply:
All initramfs changes are dropped since they are being covered in #6756

Reply:
Makes sense, but they were not actually dropped.
I see lvm2 and cryptsetup added as part of the PR.  Mind to remove those?